dev-libs/sway/metadata.xml

```diff
@@ -22,19 +26,29 @@
 		something else.
 	</longdescription>
 	<use>
-		<flag name="gdk-pixbuf">
-			Enable additional image format support via the
-			<pkg>x11-libs/gdk-pixbuf</pkg> library.
-		</flag>
-		<flag name="swaybar">Build 'swaybar': sway's status bar component.</flag>
-		<flag name="swaybg">Build 'swaybg': allows to set a desktop background image.</flag>
-		<flag name="swaygrab">Build 'swaygrab': allows taking screenshots or screencasts.</flag>
-		<flag name="swaylock">Build 'swaylock': sway's screen locker.</flag>
-		<flag name="swaymsg">Build 'swaymsg': tool for communicating with a running sway instance.</flag>
-		<flag name="wallpapers">Install sway's default wallpaper image.</flag>
-		<flag name="tray">Enable support for StatusNotifierItem tray specification.</flag>
+		<flag name="bash-completion" restrict="&gt;=dev-libs/sway-1.0_beta1">Enable bash completion support</flag>
```

Comment:
Please remove all restrict attributes. They are all superfluous unless you have a conflicting USE flag description (same flag, different meaning between versions).